assets/src/components/activities/multi_input/MultiInputDelivery.tsx

```diff
@@ -344,6 +345,7 @@ export const MultiInputComponent: React.FC = () => {
           }
           attemptState={uiState.attemptState}
           context={writerContext}
+          partOrder={orderedPartIds(model)}
```

Comment:
`orderedPartIds` gets called on every render here, which is not needed.  This should be run one time and the results stored.

Reply:
OK, done.